lib/active_merchant/billing/gateways/fortis.rb

```diff
@@ -0,0 +1,331 @@
+        1540 => 'Setup Issue, contact Support',
+        1541 => 'Device is not signature capable',
+        1588 => 'Data could not be de-tokenized',
+        1599 => 'Other Reason',
+        1601 => 'Generic Decline',
+        1602 => 'Call',
+        1603 => 'No Reply',
+        1604 => 'Pickup Card - No Fraud',
+        1605 => 'Pickup Card - Fraud',
+        1606 => 'Pickup Card - Lost',
+        1607 => 'Pickup Card - Stolen',
+        1608 => 'Account Error',
+        1609 => 'Already Reversed',
+        1610 => 'Bad PIN',
+        1611 => 'Cashback Exceeded',
+        1612 => 'Cashback Not Available',
+        1613 => 'CID Error',
+        1614 => 'Date Error',
+        1615 => 'Do Not Honor',
+        1616 => 'NSF',
+        1618 => 'Invalid Service Code',
+        1619 => 'Exceeded activity limit',
+        1620 => 'Violation',
+        1621 => 'Encryption Error',
+        1622 => 'Card Expired',
+        1623 => 'Renter',
+        1624 => 'Security Violation',
+        1625 => 'Card Not Permitted',
+        1626 => 'Trans Not Permitted',
+        1627 => 'System Error',
+        1628 => 'Bad Merchant ID',
+        1629 => 'Duplicate Batch (Already Closed)',
+        1630 => 'Batch Rejected',
+        1631 => 'Account Closed',
+        1632 => 'PIN tries exceeded',
+        1640 => 'Required fields are missing (ACH only)',
+        1641 => 'Previously declined transaction (1640)',
+        1650 => 'Contact Support',
+        1651 => 'Max Sending - Throttle Limit Hit (ACH only)',
+        1652 => 'Max Attempts Exceeded',
+        1653 => 'Contact Support',
+        1654 => 'Voided - Online Reversal Failed',
+        1655 => 'Decline (AVS Auto Reversal)',
+        1656 => 'Decline (CVV Auto Reversal)',
+        1657 => 'Decline (Partial Auth Auto Reversal)',
+        1658 => 'Expired Authorization',
+        1659 => 'Declined - Partial Approval not Supported',
+        1660 => 'Bank Account Error, please delete and re-add Token',
+        1661 => 'Declined AuthIncrement',
+        1662 => 'Auto Reversal - Processor cant settle',
+        1663 => 'Manager Needed (Needs override transaction)',
+        1664 => 'Token Not Found: Sharing Group Unavailable',
+        1665 => 'Contact Not Found: Sharing Group Unavailable',
+        1666 => 'Amount Error',
+        1667 => 'Action Not Allowed in Current State',
+        1668 => 'Original Authorization Not Valid',
+        1701 => 'Chip Reject',
+        1800 => 'Incorrect CVV',
+        1801 => 'Duplicate Transaction',
+        1802 => 'MID/TID Not Registered',
+        1803 => 'Stop Recurring',
+        1804 => 'No Transactions in Batch',
+        1805 => 'Batch Does Not Exist'
+      }
+
+      def initialize(options = {})
+        requires!(options, :user_id, :user_api_key, :developer_id, :location_id)
+        super
+      end
+
+      def authorize(money, payment, options = {})
+        commit path(:authorize, payment_type(payment)), auth_purchase_request(money, payment, options)
```

Comment:
Just a minor question. Any reason for not putting parenthesis in the commit method call? Is this personal preference or something else?